README_ESPRESSO.md

```diff
@@ -46,10 +46,72 @@ Finally, install all the dependencies:
 > mise install
 ```
 
-*
+### Install Espresso go cryptographic library
+
+This step is only needed if you use Mises as Nix automatically installs the Espresso go cryptographic library.
+
+- Create a local directory for later use. Note it has to be created under the home directory by default.
+
+  ```bash
+  cd ..
+  mkdir -p ~/local-lib
+  ```
+
+- Get `libespresso_crypto_helper.a`, by either method below.
+  - Get it from the CI. See https://github.com/EspressoSystems/espresso-network-go/releases
+    - Download `libespresso_crypto_helper-x86_64-apple-darwin.a` (or the one for linux).
+    - Move the downloaded file to `local-lib`.
+
+  - Build it locally.
+    - Download the sequencer Go code via `https://github.com/EspressoSystems/espresso-network-go/archive/refs/tags/v0.0.34.tar.gz`.
+      - Replace the version number if there’s a newer one.
+    - Go to the downloaded folder.
+
+      ```bash
+      cd espresso-network-go-0.0.34
+      ```
+
+    - Build the verification code.
+      - Make sure to not run this in the nix shell. Otherwise, the generated file will be in the wrong directory, which will cause `just` to fail later.
+      - This may require `rustup update` if the Rust version is older than expected.
+
+      ```bash
+      cargo build --release --locked --manifest-path ./verification/rust/Cargo.toml
+      ```
+
+    - Copy the `libespresso_crypto_helper.a` file.
+      - Linux:
+
+        ```bash
+        sudo cp ./espresso-network-go-0.0.34/verification/rust/target/release/libespresso_crypto_helper.a ~/local-lib/libespresso_crypto_helper-x86_64-unknown-linux-gnu.a
+        ```
+
+      - Mac:
+
+        ```bash
+        sudo cp ./espresso-network-go-0.0.34/verification/rust/target/release/libespresso_crypto_helper.a ~/local-lib/libespresso_crypto_helper-x86_64-apple-darwin.a
+        ```
```

Comment:
I originally added this section before having the "Get it from the CI" alternative, and now that I think about it, maybe we don't need it at all. Would it be better if we all just get `libespresso_crypto_helper.a` from the CI? In this way, we just have to make sure the CI has the correct file.